templates/database/database-ss.yaml

```diff
@@ -140,6 +140,10 @@ spec:
       name: "database-data"
       labels:
 {{ include "harbor.labels" . | indent 8 }}
+      annotations:
+      {{- range $key, $value := $chartmuseum.annotations }}
```

Comment:
$chartmuseum.annotations  => $database.annotations

Reply:
Yes I have corrected the changes